src/pineko/evolve.py

```diff
@@ -101,12 +101,16 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     operators_card = copy.deepcopy(default_card)
     x_grid, _pids, _mur2_grid, muf2_grid = pineappl_grid.axes()
     sv_method = sv_scheme(tcard)
-    xif = 1.0 if sv_method == "expanded" else tcard["XIF"]
+    xif = (
+        1.0
+        if (sv_method == "expanded" or sv_method == "exponentiated")
```

Comment:
Pylint is complaining https://github.com/NNPDF/pineko/actions/runs/4405224579/jobs/7715838427#step:10:27